photon-ml/src/integTest/scala/com/linkedin/photon/ml/avro/ModelProcessingUtilsTest.scala

```diff
@@ -116,7 +178,7 @@ class ModelProcessingUtilsTest extends SparkTestUtils with TestTemplateWithTmpDi
       val colEffectType = "colEffectType"
 
       // Generate the random matrix
-      val random = new Random(MathConst.RANDOM_SEED)
+      val random = new java.util.Random(MathConst.RANDOM_SEED)
```

Comment:
I think your IDE replaced the Java `Random` with the Scala `Random` in the imports, but never uses the Scala `Random`.